docs/first_config.md

```diff
@@ -26,14 +23,32 @@ A `Tiltfile` uses a small subset of Python called
 [Starlark](https://github.com/bazelbuild/starlark#tour). Most Python editors
 will work well for editing Tiltfiles.
 
-- `def oneup():`
+- `docker_build('gcr.io/windmill-test-containers/integration/oneup', '.')`
 
-The `def` line defines a function. Every time `tilt` builds your server, it
-runs this function. `Tiltfile` functions can even call `print()` to help with debugging.
+This line build the image and assigns a tag to it. The steps to build the image are in the
+`Dockerfile`.
+
+When we're doing local development, it doesn't matter that much what the image tag
+is, as long as it matches the name in our Kubernetes YAML.
 
-- `yaml = read_file('oneup.yaml')`
+(When we're doing remote development, the image tag is a URL that tells the cluster where to upload
+and download your image).
+
+If we open the Dockerfile, we see
 
-The next line reads a Kubernetes YAML. Tilt tracks while files you read, then automatically
+```
+FROM golang:1.11
+WORKDIR /go/src/github.com/windmilleng/integration/oneup
+ADD . .
+RUN go install github.com/windmilleng/integration/oneup
+ENTRYPOINT /go/bin/oneup
+```
+
+If you don't know Go, that's OK. These are steps to run to build a Go server.
+
+- `k8s_resource('oneup', 'oneup.yaml')`
+
+This next line reads a Kubernetes YAML, gives it a name and creates it in Kubernetes. Tilt tracks while files you read, then automatically
```

Comment:
s/"a Kubernetes YAML"/"Kubernetes YAML"/

I also think you can change "Tilt tracks while files you read" to "Tilt tracks dependencies; you can edit yaml or Dockerfile or Tiltfile and Tilt will ..."